packages/webcert/src/components/header/WebcertHeaderUser.tsx

```diff
@@ -40,15 +40,8 @@ const WebcertHeaderUser: React.FC = () => {
     return (
       <Wrapper>
         <UserWrapper>
-          <span>{user.name}</span>
-          <UserRole>
-            {user.role}
-            {user.protectedPerson && (
-              <>
-                {' - '} <ProtectedPersonDoctorModal />
-              </>
-            )}
-          </UserRole>
+          <span>{`${user.name} - ${user.role}`}</span>
+          <UserRole>{user.protectedPerson && <ProtectedPersonDoctorModal />}</UserRole>
```

Comment:
Why is the wrapper called UserRole if it only includes the protected person modal? Also, isn't this gonna result in an empty tag if the user doesn't have the protected person flag?

Reply:
Good question, I don't know why it's called UserRole but I changed it to StyledSpan and moved the condition outside of the  span.